docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst

```diff
@@ -0,0 +1,1204 @@
+
+After loading the workflow environment, users may continue to :numref:`Section %s <ExptConfig>` for instructions on setting the experiment configuration parameters.
+
+.. _Load-WF-L234:
+
+Loading the Workflow Environment on Level 2-4 Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users on non-Level 1 systems will need to create a conda workflow environment, modify a ``wflow_*`` file to reflect the location of required modules, and load the workflow modules using the modified ``wflow_*`` file. 
+
+Create a *conda* Workflow Environment
+```````````````````````````````````````
+
+.. note::
+    Examples in this subsection presume that the user is running in the Terminal with a bash shell environment. If this is not the case, users will need to adjust the commands to fit their command line application and shell environment. 
+
+.. _MacMorePackages:
+
+MacOS ONLY: Install/Upgrade Mac-Specific Packages
+"""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. attention:: 
+
+   This subsection is for Mac OS users only. Users on Linux systems can skip to :ref:`Creating the workflow_tools Environment on Linux and Mac OS <LinuxMacVEnv>` for instructions.
+
+
+MacOS requires the installation of a few additional packages and, possibly, an upgrade to bash. Users running on MacOS should execute the following commands:
+
+.. code-block:: console
+
+   bash --version
+   brew install bash       # or: brew upgrade bash
+   brew install coreutils
+   brew gsed               # follow directions to update the PATH env variable
+
+
+.. _LinuxMacVEnv: 
+
+Creating the ``workflow_tools`` Environment on Linux and Mac OS
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+On generic Mac and Linux systems, users need to create a conda ``workflow_tools`` environment. The environment can be stored in a local path, which could be a default location or a user-specified location (e.g., ``$HOME/condaenv/venvs/`` directory). (To determine the default location, use the ``conda info`` command, and look for the ``envs directories`` list.) The following is a brief recipe for creating a virtual conda environment on non-Level 1 platforms. It uses the aarch64 (64-bit ARM) Miniforge for Linux and installs into $HOME/conda. Adjust as necessary for your target system.
+
+.. code-block:: console
+
+   wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh
+   bash Miniforge3-Linux-aarch64.sh -bfp ~/conda
+   rm Miniforge3-Linux-aarch64.sh
+   source ~/conda/etc/profile.d/conda.sh
+   conda activate
+   conda install -y conda-build conda-verify
+   cd path/to/your/workflow-tools/clone
+   conda build recipe
+   conda create -y -n workflow_tools -c local uwtools
+   conda activate workflow_tools
```

Comment:
@christinaholtNOAA Can you suggest changes here to make this work for the SRW? I switched some occurrences of `uwtools` to `workflow_tools`, but I don't know if what I did is correct and/or if the one I left should also be switched.

Reply:
The one `uwtools` -> `workflow_tools` on Line 207 below is the only one I see at this point.